documentation/bolt_configuration_reference.md.erb

```diff
@@ -108,10 +108,10 @@ Transport configuration options can be set in both the configuration file and in
 <% @transports[:options].each do |transport, options| %>
 ### <%= transport %>
 
-| Option | Description | Default |
-| ------ | ----------- | ------- |
-<% options.each do |option, desc| -%>
-| `<%= option %>` | <%= desc %> | <%= @transports[:defaults][transport][option].to_s || "" %> |
+| Option | Description | Type | Default |
+| ------ | ----------- | ---- | ------- |
+<% options.each do |option, data| -%>
+| `<%= option %>` | <%= data[:desc] %> | <%= data[:type] %> | <%= @transports[:defaults][transport][option].to_s || "" %> |
```

Comment:
While I do like having the option's type listed, it does look a little weird when types like `Boolean` are missing.